internal/plugin/service/kafka/models.go

```diff
@@ -0,0 +1,38 @@
+package kafka
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type kafkaModel struct {
+	Timeouts              timeouts.Value `tfsdk:"timeouts"`
+	ID                    types.String   `tfsdk:"id"`
+	AdditionalDiskSpace   types.String   `tfsdk:"additional_disk_space"`
```

Comment:
consider changing `AdditionalDiskSpace` to `types.Int64` if it represents a numeric value in megabytes or gigabytes